lte/gateway/python/magma/mobilityd/dhcp_client.py

```diff
@@ -278,10 +278,10 @@ def _process_dhcp_pkt(self, packet, state: DHCPState):
 
                 dhcp_router_opt = self._get_option(packet, "router")
                 if dhcp_router_opt is not None:
-                    router_ip_addr = ip_address(dhcp_router_opt)
+                    router_ip_addr = str(ip_address(dhcp_router_opt))
```

Comment:
This makes little sense. Why convert from (presumably) str to ip back to str?

Reply:
Both `dhcp_router_opt` and `dhcp_server_ip` come from scapy's DHCP.options and are indeed `str` (https://github.com/secdev/scapy/blob/b3726ca2bf2b0e7d4999c6165807a7a314bf962f/scapy/layers/dhcp.py#L207).
As the `router_ip_addr` is `str` in `self.dhcp_gw_info.update_ip` and `DHCPDescriptor` any convertion is not needed.

Reply:
Thanks for looking into that! :+1: